tests/infra_test.go

```diff
@@ -1566,7 +1566,7 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 				_, err = virtClient.ClusterProfiler().Dump(&v1.ClusterProfilerRequest{})
 				Expect(err).ToNot(BeNil())
 			})
-			It("[QUARANTINE] is enabled it should allow subresource access", func() {
+			It("is enabled it should allow subresource access", func() {
```

Comment:
Did the test pass the quarantine after a fix? 
We only remove quarantine when there is a fix and 2 weeks with successful runs have passed. I do not see the fix or the dates mentioned in PR description.

Reply:
Yeah, I somehow included @enp0s3 's commit 98f39da1a8 in my PR.